spec/services/processors/jira_project_development_cycle_updater_spec.rb

```diff
@@ -75,6 +75,28 @@
         subject
         expect(last_issue.resolved_at).to be_nil
       end
+
+      context 'when the issue type is not in the enum' do
+        let(:bugs) do
+          [
+            {
+              'key': 'TES-4',
+              'fields': {
+                'customfield_10000': [{ 'value': 'production' }],
+                'created': informed_at_date,
+                'resolutiondate': resolved_at_date,
+                'issuetype': {
+                  'name': 'test_execution'
+                }
+              }
+            }
+          ]
+        end
+
+        it 'does not creates a record on the db' do
```

Comment:
"_on_ the db" sounds a bit off to me. Found [this discussion](https://forum.wordreference.com/threads/on-in-into-a-database.3118373/) about in/on/into db.